### PR TITLE
Fixes Relative Paths pointing to Resources

### PR DESCRIPTION
**Pull request by juanpedromoreno**
Please, @rafaparadela have a look. Thanks.

### CODE REVIEW

File: src/main/scala/microsites/layouts/Layout.scala [hunk 1]
```diff
@@ -18,16 +18,16 @@ trait Layout {
     meta(name := "viewport", content := "width=device-width, initial-scale=1.0"),
     meta(name := "description", content := config.description),
     meta(name := "author", content := config.author),
-    meta(name := "og:image", content := "img/poster.png"),
+    meta(name := "og:image", content := "/img/poster.png"),
```
Comment (rafaparadela) [inline src/main/scala/microsites/layouts/Layout.scala]: We need absolute url here

File: src/main/scala/microsites/layouts/Layout.scala [hunk 2]
```diff
@@ -18,16 +18,16 @@ trait Layout {
...
     meta(name := "og:title", content := config.name),
     meta(name := "og:site_name", content := config.name),
     meta(name := "og:url", content := config.homepage),
     meta(name := "og:type", content := "website"),
     meta(name := "og:description", content := config.description),
-    meta(name := "twitter:image", content := "img/poster.png"),
+    meta(name := "twitter:image", content := "/img/poster.png"),
```
Comment (rafaparadela) [inline src/main/scala/microsites/layouts/Layout.scala]: We need absolute url here

File: src/main/scala/microsites/layouts/Layout.scala [hunk 3]
```diff
@@ -18,16 +18,16 @@ trait Layout {
...
     meta(name := "twitter:card", content := "summary_large_image"),
     meta(name := "twitter:site", content := config.twitter),
-    link(rel := "icon", `type` := "image/png", href := "img/favicon.png"))
+    link(rel := "icon", `type` := "image/png", href := "/img/favicon.png"))
```
Comment (rafaparadela) [inline src/main/scala/microsites/layouts/Layout.scala]: We need absolute url here

**Comment by israelperezglez** [comment]
LGTM!

**Pull request merged**